libyul/optimiser/FullInliner.cpp

```diff
@@ -235,7 +235,15 @@ Statement BodyCopier::operator()(VariableDeclaration const& _varDecl)
 Statement BodyCopier::operator()(FunctionDefinition const& _funDef)
 {
 	assertThrow(false, OptimizerException, "Function hoisting has to be done before function inlining.");
-	return _funDef;
```

Comment:
The BodyCopier is not designed to copy functions (they also do not need to be copied, they can just be referenced). This line is just here to prevent the compiler warning. Something like `return {}` could work. If not, just take the simplest variant of statement.